src/BugsnagUnity/Configuration.cs

```diff
@@ -60,6 +63,13 @@ protected virtual void SetupDefaults(string apiKey)
     public virtual bool AutoCaptureSessions { get; set; }
 
     public virtual LogTypeSeverityMapping LogTypeSeverityMapping { get; } = new LogTypeSeverityMapping();
+
+    public virtual bool DetectAnrs { get; set; }
+
+    public virtual long AnrThresholdMs {
+      get => _anrThresholdMs;
+      set => _anrThresholdMs = value < 1000 ? 1000 : value;
```

Comment:
Why is this logic duplicated here? Presumably the native android implementation already implements this.